CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ but cannot always guarantee backwards compatibility. Changes that may **break co
 **Improved**
 - Added `IQRDetector`, that allows to detect anomalies using the interquartile range algorithm. [#2441] by [Igor Urbanik](https://github.com/u8-igor).
 - Made README's forecasting model support table more colorblind-friendly. [#2433](https://github.com/unit8co/darts/pull/2433)
+- Allowed passing of kwargs to the `fit` functions of `Prophet` and `AutoARIMA`
+- 🔴 Restructured the signatures of `ExponentialSmoothing` `__init__` and `fit` functions so that the passing of additional parameters is consistent with other models
+  - Keyword arguments to be passed to the underlying model's constructor must now be passed as keyword arguments instead of a `dict` to the `ExponentialSmoothing` constructor
+  - Keyword arguments to be passed to the underlying model's `fit` function must now be passed to the `ExponentialSmoothing.fit` function instead of the constructor
```

Comment:
Once all suggestions have been addressed, we could formulate it as below :) 

```suggestion
- Improvements to `ForecastingModel` : [#2460](https://github.com/unit8co/darts/pull/2460) by [DavidKleindienst](https://github.com/DavidKleindienst).
  - All forecasting models now support keyword arguments `**kwargs` when calling `fit()` that will be passed to the underlying model's fit function.
  - 🔴 Changes to `ExponentialSmoothing` for a unified API:
    - Removed `fit_kwargs` from `__init__()`. They must now be passed as keyword arguments to `fit()`.
    - Parameters to be passed to the underlying model's constructor must now be passed as keyword arguments (instead of an explicit `kwargs` parameter).
```